keyboards/kinesis/stapelberg_i2c/stapelberg_i2c.c

```diff
@@ -0,0 +1,61 @@
+#include "stapelberg_i2c.h"
+
+void matrix_init_kb(void) {
+	// put your keyboard start-up code here
+	// runs once when the firmware starts up
+    // * Set our LED pins as output
+    DDRF |= (1<<0); // Keypad LED
```

Comment:
Could these be updated to use the GPIO Controls? 
```suggestion
    setPinOutput(F0); // Keypad LED
```